Source/WebCore/platform/graphics/gstreamer/AudioTrackPrivateGStreamer.h

```diff
@@ -60,9 +60,10 @@ class AudioTrackPrivateGStreamer final : public AudioTrackPrivate, public TrackP
     AtomString label() const final { return m_label; }
     AtomString language() const final { return m_language; }
 
+    void updateConfigurationFromCaps(const GRefPtr<GstCaps>&&) final;
```

Comment:
Wow, I guess you just moved the function, but const && usually makes no sense. If I reviewed this in the past I apologize for missing this. You either pass
* const &:  to state that only const operations or a copy should be made
* non const &&: to state that what you are passing is the full ownership of the object.

Please change this, everywhere. No const &&, please. If it's too much for this patch, please do it before in a separate patch.